koku/masu/external/downloader/azure_local/azure_local_report_downloader.py

```diff
@@ -106,6 +102,8 @@ def download_file(self, key, stored_etag=None, manifest_id=None, start_date=None
             (String): The path and file name of the saved file
 
         """
+        file_names = []
+        date_range = {}
         local_filename = utils.get_local_file_name(key)
```

Comment:
Do these need to be assigned here? I don't see them used before they are assigned again on line 121.

Reply:
Could be dropped but again local downloader so not that concerned.